src/index.html

```diff
@@ -152,18 +176,67 @@ <h2 class="modal__title">New post</h2>
         </form>
       </div>
     </div>
+
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
         <button
           aria-label="Close button"
           type="button"
           class="modal__close modal__close_type_preview"
         ></button>
+
         <img class="modal__image" src="" alt="" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/validation.js"></script>
-    <script defer src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__content">
+        <button
+          aria-label="Close modal"
+          type="button"
+          class="modal__close"
+        ></button>
+
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form class="modal__form" id="profile-avatar-form" novalidate>
+          <label class="modal__label" for="profile-avatar-form">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="url"
+              class="modal__input"
+              placeholder="avatar goes here"
+              name="avatar"
+              required
+            />
+            <span id="profile-avatar-input-error" class="modal__error"></span>
+          </label>
+          <button type="submit" id="modal__button-error" class="modal__button">
```

Comment:
`id="modal__button-error"` is already in line 121, so you should change the `id` here

* It's necessary to fix all the errors (only **errors**, not warnings) found by html `validator`: https://validator.w3.org/nu/#textarea . Just copy your HTML code to the text area and hit `Check`. This validator will help you find and fix small typos, syntax errors and other similar issues. This short **video** shows how to use validators for your code: https://youtu.be/Yc_Qw4-oVq0